<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="plus2Button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plus1Button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plus3Button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodge/Burn Amount:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
